.github/workflows/build_linux_wheels.yaml

```diff
@@ -0,0 +1,175 @@
+# Heavy inspiration taken from:
+# https://github.com/pytorch/test-infra/blob/main/.github/workflows/build_wheels_linux.yml
+
+name: Build Linux Wheels
+
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+    tags:
+        # NOTE: Binary build pipelines should only get triggered on release candidate builds
+        # Release candidate tags look like: v1.11.0-rc1
+        - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+  workflow_dispatch:
+
+
+permissions:
+  id-token: write
+  contents: read
+
+
+jobs:
+  setup-variables:
+    runs-on: ubuntu-latest
+    outputs:
+      TEST_INFRA_REPOSITORY: pytorch/test-infra
+      TEST_INFRA_REF: main
+      REPOSITORY: pytorch/torchtune
+      REF: pkg-for-wheel
+      PACKAGE_NAME: torchtune
+      PACKAGE_TYPE: wheel
+      ARCH: x86_64
+      SMOKE_TEST_SCRIPT: ""
+    steps:
+      - run: echo "Exposing env vars to all jobs"
+  generate-matrix:
```

Comment:
Any value to just adding a comment at the top of this job about which versions we should be building so folks don't have to go to the test-infra yaml to find this stuff?